src/app/components/educational-programs/educational-programs.component.ts

```diff
@@ -1,69 +1,103 @@
 import { Component, OnInit } from '@angular/core';
-import { FormBuilder, FormGroup, FormArray } from '@angular/forms';
-import { Subscription } from 'rxjs';
+
+export interface EducationalProgram {
+  name: string;
+  acceptanceFrom: string;
+  acceptanceTo: string;
+  programFrom: string;
+  programTo: string;
+}
+
+const ELEMENT_DATA: EducationalProgram[] = [
```

Comment:
Not critical, but I suggest move mock data and interfaces away from component to separate files